packages/matter.js/src/cluster/ClusterIdentifier.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// There will be an enum here.  But should ClusterIdentifier be this enum?
```

Comment:
I would do this as enum with all „cluster ids defined by specs“ irrelevant if we have them already implemented or not. With this they are not limiting in any way - at least not more limiting then the specs itself ;-) or what do you mean? 
i would not rename ClusterId because that’s official from spec type „cluster-id“ like the others too. In theory